source/basic.tex

```diff
@@ -391,7 +391,7 @@
 \item an object of type \tcode{T} is defined~(\ref{basic.def}), or
 \item a non-static class data member of type \tcode{T} is
 declared~(\ref{class.mem}), or
-\item \tcode{T} is used as the object type or array element type in a
+\item \tcode{T} is used as the object's type or array element type in a
```

Comment:
It seems a little awkward to use the possessive in only one of the two cases here. Maybe "used as the allocated type or array element type" would be better?